docs/source/implementations/backtest_statistics.rst

```diff
@@ -146,59 +155,86 @@ Snippet 14.4, page 201, Calculates drawdowns and time under water
 
 The function returns two series:
 
-1.Drawdown series index is time of a high watermark and value of a drawdown after
-
+1.Drawdown series index is time of a high watermark and the drawdown value.
 
-2.Time under water index is time of a high watermark and how much time passed till next high watermark in years.
+2.Time under water index is time of a high watermark and how much time passed till next high watermark is reached, in years. Also includes time between the last high watermark and last observation in returns as the last Time under water element. Without this element the estimations of Time under water can be biased.
 
 
 .. function:: drawdown_and_time_under_water(returns: pd.Series, dollars: bool = False) -> tuple:
 
     :param returns: (pd.Series) returns from bets
-    :param dollars: (bool) flag if given dollar performance and not returns
+    :param dollars: (bool) flag if given dollar performance and not returns.
+                    If dollars, then drawdowns are in dollars, else as a %.
     :return: (tuple of pd.Series) series of drawdowns and time under water
-                                if dollars, then in dollars, else as a %
 
 An example showing how Drawdown and Time Under Water function is used with account data in dollars::
 
 	from mlfinlab.backtest_statistics import drawdown_and_time_under_water
 
-	drawdown, tuw = drawdown_and_time_under_water(returns, dollars = True)
+	drawdown, tuw = drawdown_and_time_under_water(returns, dollars=True)
+
+Information Ratio
+==============================
+
+Calculates Annualized Information Ratio for a given pandas Series of normal or log returns.
+
+It is the annualized ratio between the average excess return and the tracking error. The excess return is measured as the portfolio’s return in excess of the benchmark’s return. The tracking error is estimated as the standard deviation of the excess returns.
+
+Benchmark should be provided as a return for the same time period as that between input returns. For example, for the daily observations it should be the benchmark of daily returns.
+
+Calculated as:
 
-Annual Sharpe Ratio
+        :math:`SharpeRatio = \frac{E[Returns - Benchmark]}{\sqrt{V[Returns - Benchmark]}} * \sqrt{n}`
+
+.. function:: information_ratio(returns: pd.Series, benchmark: float = 0,
+                                entries_per_year: int = 252) -> float:
+
+    :param returns: (pd.Series) returns - normal or log
+    :param benchmark: (float) benchmark for performance comparison (0 by default)
+    :param entries_per_year: (int) times returns are recorded per year (daily by default)
+    :return: (float) Annualized Information Ratio
+
+An example showing how Annualized Information Ratio function is used with monthly cumulative returns data::
+
+	from mlfinlab.backtest_statistics import information_ratio
+
+	information_r = information(returns, benchmark=0.005, entries_per_year=12)
```

Comment:
This function cant be called information, it should be called information_ratio.

Reply:
Will change